modules/ROOT/pages/afm-edit-test.adoc

```diff
@@ -0,0 +1,23 @@
+= Create a New Version of an Existing Test
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You cannot edit a test that exists for a monitor, but you can create a new version of one. In the new version, you can change, remove, or add assertions, headers, endpoints, and reporters.
+
+You can create a new version of a test only if the test has not been customized to include features that the *Create a Monitor from Scratch* page does not support. (It is possible to customize a test by downloading it, editing it manually, and then uploading it.) Tests created manually and then uploaded also are not editable if they contain features that the *Create a Monitor from Scratch* page does not support.
+
+== Procedure
```

Comment:
Other documents that I am currently editing do not separate the steps from the text that introduces them.
I'm thinking DITA: short description, steps, result.
But I see that the result is quite lengthy, so...maybe this format is for the best....

Reply:
@kbostrom , separating them makes topics easier to scan, I think.